index.js

```diff
@@ -1,15 +1,31 @@
 'use strict';
+
 var fs = require('fs');
 var path = require('path');
 var crypto = require('crypto');
+var gulp = require('gulp');
```

Comment:
Which in turn is a dependency of gulp. I don't see why that dependency is better than the gulp dependency. Especially since this is a gulp plugin.